wagtail/admin/wagtail_hooks.py

```diff
@@ -518,3 +548,48 @@ def register_core_features(features):
             'entity_decorators': {'LINK': link_entity}
         }
     })
+    features.register_editor_plugin(
+        'draftail', 'superscript', draftail_features.InlineStyleFeature({
+            'type': 'SUPERSCRIPT',
+            'icon': 'cog',
+            'description': ugettext('Superscript'),
+        })
+    )
+    features.register_converter_rule('contentstate', 'superscript', {
+        'from_database_format': {
+            'sup': InlineStyleElementHandler('SUPERSCRIPT'),
+        },
+        'to_database_format': {
+            'style_map': {'SUPERSCRIPT': 'sup'}
+        }
+    })
+    features.register_editor_plugin(
+        'draftail', 'subscript', draftail_features.InlineStyleFeature({
+            'type': 'SUBSCRIPT',
+            'icon': 'cog',
+            'description': ugettext('Subscript'),
+        })
+    )
+    features.register_converter_rule('contentstate', 'subscript', {
+        'from_database_format': {
+            'sub': InlineStyleElementHandler('SUBSCRIPT'),
+        },
+        'to_database_format': {
+            'style_map': {'SUBSCRIPT': 'sub'}
+        }
+    })
+    features.register_editor_plugin(
+        'draftail', 'strikethrough', draftail_features.InlineStyleFeature({
+            'type': 'STRIKETHROUGH',
+            'icon': 'cog',
+            'description': ugettext('Strikethrough'),
+        })
+    )
+    features.register_converter_rule('contentstate', 'strikethrough', {
+        'from_database_format': {
+            's': InlineStyleElementHandler('STRIKETHROUGH'),
+        },
+        'to_database_format': {
+            'style_map': {'STRIKETHROUGH': 's'}
+        }
+    })
```

Comment:
I think this is missing inline `code` formatting?

Reply:
Do you mean a new identifier called `inline-code`?